typescript_sdk/packages/reality-capture/src/service/data_handler.ts

```diff
@@ -0,0 +1,305 @@
+import * as fs from "fs";
+import * as path from "path";
+import { ContainerClient } from "@azure/storage-blob";
+import { AuthorizationClient } from "@itwin/core-common";
+import { BentleyError } from "@itwin/core-bentley";
+//TODO : ThreadPool type does not have native Node.js equivalent. Multi-thread for IO must be readapted (here, ploads and downloads are sequential.
+//import { ThreadPool } from "some-threadpool-lib";
```

Comment:
fyi: 
- [piscina](https://www.npmjs.com/package/piscina)
- [tinypool](https://www.npmjs.com/package/tinypool)

Reply:
i hope you guys have an issue tracking this, and will remember these solutions above